extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/ContentAndItagItemAndIsUrl.java

```diff
@@ -0,0 +1,15 @@
+package org.schabi.newpipe.extractor.services.youtube;
+
+public class ContentAndItagItemAndIsUrl {
+    public String content;
+    public ItagItem itagItem;
+    public boolean isUrl;
+
+    public ContentAndItagItemAndIsUrl(final String content,
+                                      final ItagItem itagItem,
+                                      final boolean isUrl) {
+        this.content = content;
+        this.itagItem = itagItem;
+        this.isUrl = isUrl;
+    }
+}
```

Comment:
Is there a better name I can use for this class?

Reply:
TemporaryStreamInfo?

Reply:
ItagInfo?

Reply:
`ItagInfo` is better than `TemporaryStreamData`, so I think I will use this name.